.travis.yml

```diff
@@ -28,7 +28,8 @@ script:
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_lldp_neighbors_detail
 - nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_arp_table
-- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_peers
+- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
+#- nosetests -v TestJunOSDriver:TestGetterJunOSDriver.test_get_ntp_stats
```

Comment:
Seem that all of them have the same line, I will not comment on them to avoid spamming you even more but I would appreaciate if you could remove them : )